helm/hpcc/values.schema.json

```diff
@@ -426,7 +426,19 @@
         "defaultSprayParts" : {
             "description": "Number of parts sprayed by default",
             "type" : "integer"
+        },
+        "cost" : {
+            "description": "Costs associated with the storage and use of the plane",
```

Comment:
trivial/formatting: inconsistent indentation, 2 space indenting used normally in this file - only other exception I think is 'defautSprayParts'